### PR TITLE
Removed channel and probe from required fields in imaging_protocol. Fixes #1430

### LINKED ISSUE #1430: Make imaging_protocol 'channel' and 'probe' optional
**For which schema is a change/update being suggested?**
I would like to request an update to the 'type/protocol/imaging/imaging_protocol.json' schema

**What should the change/update be?**
Removing the two module fields 'channel' and 'probe' which are part of the 'imaging_protocol' schema from the 'required' array. 

This update constitutes a Minor change to the schema(s) it affects.

**Why is the change requested?**
The 'channel' and 'probe' imaging modules are not applicable for some imaging technologies (such as Visium). Currently,  Visium datasets are blocked as they fail validation. Making the two module fields optional would allow datasets to be unblocked and pass validation.

*e.g.*: Data contributors need to be able to assign a random integer to a cell line in order to keep track of it.

Datasets being blocked: 
AdultLungElo
E-MTAB-10386
Spatial multi-omic map of human myocardial infarction

### PR DESCRIPTION
**Pull request by willrockout**
### Release notes

For `imaging_protocol.json` schema:
- Removed `channel` and `probe` from the required field list.
 

### Reviews requested

- Need 2 Reviewers to approve because this is a minor update
- Last Reviewer to review/approve, merge changes

### CODE REVIEW

**Review by theathorn — APPROVED** [review]

**Comment by gabsie** [comment]
Hey, everyone, can we possibly get these reviews in, possibly by Monday next week? @aherbst-broad , @amnonkhen @hannes-ucsc , @khajoue2 , @kishorikonwar , @NoopDog , @ruchim ?

Thank you!

**Review by ruchim — APPROVED** [review]
Looks good! This question doesn’t bar any changes — tho if someone can do a quick explanation of why these are no longer required/ what’s the new info we learned in #dcp-2, that would be appreciated!

**Review by hannes-ucsc — APPROVED** [review]

**Review by ESapenaVentura — APPROVED** [review]
LGTM 👍

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]
Actually I just realised it's missing the update_log.csv file

**Review by NoopDog — APPROVED** [review]

**Review by aherbst-broad — APPROVED** [review]

File: json_schema/update_log.csv [hunk 1]
```diff
@@ -1 +1,2 @@
 Schema,Change type,Change message,Version,Date
+type/protocol/imaging/imaging_protocol.json,minor,"Removed channel and probe from required fields. #Fixes 1420",,
```
Comment (ESapenaVentura) [inline json_schema/update_log.csv:2]:
```suggestion
type/protocol/imaging/imaging_protocol,minor,"Removed channel and probe from required fields. #Fixes 1420",,
```

**Review by ESapenaVentura — APPROVED** [review]

**Pull request merged**